.husky/pre-push

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Lint all staged files
+yarn nx format:check
```

Comment:
`yarn nx format:write` does the trick right?

Reply:
Yeah exactly. I run in to that enough with the current setup that I have an alias to amend the previous commit